modules/abc/20240408-yosyshq/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+"""https://www.portaudio.com/"""
+
+module(
+    name = "abc",
+    version = "20240408-yosyshq",
```

Comment:
I'd recommend using a pseudo-version (https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#pseudo-versions) in this case.

Reply:
@Wyverald so what concretely would this be? `0.0.0-20240408-yosyshq`?

Reply:
probably, yeah. normally the string after the date/time would be the commit hash, but I'm ok with using the fork name if you'd like to.

Reply:
Updated the name!